src/components/scenes/FioAddressListScene.js

```diff
@@ -54,8 +55,8 @@ export class FioAddressListScene extends Component<Props, State> {
     refreshAllFioAddresses(() => this.checkForFioAddresses())
```

Comment:
refreshAllFioAddresses does not need a callback. You already have redux state fioAddressesLoading and you can simply look at this to determine whether the addresses have been loaded and if not show a spinner

Reply:
i'm using cb to navigate to the address register scene. Changed using componentDidUpdate